### PR TITLE
Homework01_Surda

### PR DESCRIPTION
**Pull request by DanielSurda**

### CODE REVIEW

File: team/482451/montyhall.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
+from random import randint
+
+c = 0
+list2 = []
+
+while c != 100000:
```
Comment (kpetrekova) [inline team/482451/montyhall.py:6]: If the number of iterations is fixed, a for loop is preferable.

File: team/482451/montyhall.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        list[0] = 1
+    elif p == 1:
+        list[1] = 1
+    else:
+        list[2] = 1
+
+    selection = randint(0, 2)
+    list.pop(selection)
+
+    if list[0] == 0 and list[1] == 0:
```
Comment (kpetrekova) [inline team/482451/montyhall.py:21]: Redundant use of if-else. Simply check for the presence of a number 1 in any position.

File: team/482451/montyhall.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+        list2.append(0)
+
+    else:
+        list2.append(1)
+
+    c += 1
+
+print(list2 .count(1), '/', c)
```
Comment (kpetrekova) [inline team/482451/montyhall.py:29]: Check your spacing/formatting in `list2 .count(1)`

File: team/482451/montyhall.py [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+c = 0
+list2 = []
+
+while c != 100000:
+
+    p = randint(0, 2)
+    list = [0, 0, 0]
+
+    if p == 0:
```
Comment (kpetrekova) [inline team/482451/montyhall.py:11]: Unnecessary use of if-else,  just store the value 1 at position p.

File: team/482451/montyhall.py [hunk 5]
```diff
@@ -0,0 +1,30 @@
+from random import randint
+
+c = 0
+list2 = []
```
Comment (kpetrekova) [inline team/482451/montyhall.py:4]: Probably used to store intermediate results? 
No need for list. You can use int to add a win (1) or a loss (0). Then there will be no need for a count function later.

File: team/482451/montyhall.py [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+
+while c != 100000:
+
+    p = randint(0, 2)
+    list = [0, 0, 0]
```
Comment (kpetrekova) [inline team/482451/montyhall.py:9]: You don't want to name your variables the same as the types that we'll be working with.

File: team/482451/montyhall.py
Comment (kpetrekova) [inline team/482451/montyhall.py:1]: Please add comments to make the code more understandable.

**Review by kpetrekova — CHANGES_REQUESTED** [review]

File: team/482451/montyhall.py
Comment (janrygl) [reply team/482451/montyhall.py:1]: Using `please` in code reviews make them always better.

File: team/482451/montyhall.py (hunk 1 above)
Comment (janrygl) [reply team/482451/montyhall.py:6]: Good point. Always more readable.

File: team/482451/montyhall.py (hunk 6 above)
Comment (janrygl) [reply team/482451/montyhall.py:9]: Totally agree (unfortunately, many programmers break the rule for the function `id`).

File: team/482451/montyhall.py (hunk 3 above)
Comment (janrygl) [reply team/482451/montyhall.py:29]: Also always when printing multiple numbers, add description what is their semantic meaning.

File: team/482451/montyhall.py (hunk 2 above)
Comment (janrygl) [reply team/482451/montyhall.py:21]: Depending on the programmer level, we can consider showing the solution if not trivial:

`list2.append(any(list))`

**Review by janrygl — COMMENTED** [review]
@kpetrekova Professionally done, friendly code review.

**Pull request closed without merging**